.github/workflows/releaseConfigurator.yaml

```diff
@@ -0,0 +1,64 @@
+name: Configurator Release
+
+# This workflow manually triggers the automated process of release based on the changelog.
+# Is recommended to run `make release-changelog` before to see an example of what is going to be generated.
+
+# IMPORTANT No matter the branch selected when triggered, the workflow will always execute on 'main' branch.
```

Comment:
Should we check that the user is not trying to run it against a different branch and in that case fail?

I fear that at some point someone wants to release "fix/pocTESTdebug" and ends up releasing main

Reply:
yes make sense!

Reply:
done